blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -183,7 +185,7 @@ class CardList extends React.Component {
                   </Title>
                   )}
                 <div>
-                  <Title
+                  <HeadlineText
                     primaryFont={getThemeStyle(arcSite)['primary-font-family']}
```

Comment:
wouldn't these styled components need primaryFont for font family with title https://github.com/WPMedia/fusion-news-theme-blocks/pull/600/files#diff-e340cd7b2d90a0bde3c21d59f89a1679db1f6bc706c4ea7eeb1a1ee00a1d7e47R45 ?

Reply:
Sorry, I could not get your point? HeadlineText styled component accepting primaryFont as a parameter. We also using that component at line 228 as the same way. Is there anything can improve here?

Reply:
no no worries, I misread this diff, my bad